src/hermes_claude_agent_sdk/parity/v4_gateway_observer.py

```
@@ -0,0 +1,311 @@
+                try: self._call_frame(method, decision, value)
+                except V4GatewayObserverViolation: self._failed = True; raise
+                return projector(value) if projector is not None else None
+            result = self._gateway.call(method, values, timeout=timeout, projector=composed)
+            if not seen:
+                self._call_frame(method, decision, result)
+            return result
+        except V4GatewayObserverViolation:
+            self._failed = True
+            raise
+    rpc = call
+    def _event(self, value: object) -> None:
```

Ruff (0.16.3)

[warning] Too many branches (23 > 12)

(PLR0912)